.azure-ci.yml

```diff
@@ -0,0 +1,114 @@
+variables:
+  - name: _TeamName
+    value: DotNetCore
+
+jobs:
+- template: /eng/common/templates/jobs/jobs.yml
+  parameters:
+    enableMicrobuild: true
+    enablePublishBuildArtifacts: true
+    enablePublishTestResults: false
+    # TODO - set 'enablePublishBuildAssets: true' once we're publishing output to the 'artifacts' dir
+    enablePublishBuildAssets: false
+    enableTelemetry: true
+    helixRepo: dotnet/standard
+    jobs:
+    - job: Windows_NT
+      pool: 
+        # For public or PR jobs, use the hosted pool.  For internal jobs use the internal pool.
+        # Will eventually change this to two BYOC pools.
+        ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+          name: dotnet-external-temp
+        ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+          name: dotnet-internal-temp
+      variables:
+      - name: _BuildConfig 
+        value: Release
+      # Enable signing for internal, non-PR builds
+      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+        - name: _SignType
```

Comment:
nit:  I, personally, find the array syntax for variables ugly and the (positive?) side-affect is that I've been urging people to define less variables.  Consider collapsing this into something like...

```
variables:
- name: _BuildConfig
  value: Release
- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
  - name: _BuildArgs
    value: /p:SignType=Real /p:DotNetSignType=Real /p:MicroBuild_SigningEnabled=true 
      /p:DotNetPublishBlobFeedKey=$(dotnetfeed-storage-access-key-1) 
      /p:DotNetPublishBlobFeedUrl=https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
      /p:DotNetPublishToBlobFeed=true
      /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat) 
      /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
      /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)
- ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
  - name: _BuildArgs
    value: ''
```